### DISCUSSION TITLE
Startpage

### DISCUSSION THREAD

**Discussion by SystemError12** [Floorp-Projects/Floorp#684] [Feature Request]
Is it possible to add a customizable startpage feature, similar to Vivaldi or Opera?  Some also call it Speed Dial. Currently I am using the addon "[yet another speed dial](https://addons.mozilla.org/en-US/firefox/addon/yet-another-speed-dial/)" which is great, but still not as good as Opera/Vivaldi where it is properly integrated with the browser and its bookmarks and just looks better designed.

**Comment by summoner001** [comment]
Speed dial like old opera v12 FTW!

I would also like to use it in a possible Floorp app for Android.

**Comment by Telmesomething** [comment]
This suggestion is at least theoretically possible and if you're willing to experiment, then you should be able to achieve your goal of creating a custom speed dial page.

1. What you're going to do is to make use of Windows 10 or 11 multi-tasking. If case you don't know how to start with that see Microsoft's [link ](https://support.microsoft.com/en-us/windows/snap-your-windows-885a9b1e-a983-a3b1-16cd-c531795e6241)on the subject. 
2. So assuming you've spit the screen in two. What you need now is two pages of Floorp. One must be completely blank and the other, your home page with its existing links.
3. Make the blank page the one on the left hand side of the desktop and go to the site you want to create a speed dial link for. 
4. Next, click and hold down the mouse button on the left hand padlock in the URL bar and drag it to the home page on the right. This should create a shortcut to the page you have showing on the left hand side of the screen. If it works repeat to your heart's content. 

Before you begin though,  I suggest creating a Windows System Restore point so that you can go back to the time before you attempted this hack.

Good luck!

**Reply by summoner001** [reply]
Thank you! I haven't used windows in over 10 years. I use Arch Linux + Gnome DE. For this reason I will not switch to windows. I thought Floorp was going to have actual development that's why I suggested the speed dial idea.

According to them, good usable web browsers are exhausted in Opera and Vivaldi applications.

And actual development died with the old Presto Opera.

**Comment by tasinone** [comment]
Well, I have made the same suggestion. It will be great if they add a Startpage like Vivaldi to Floorp. Currently, I am using [QuickStart](https://github.com/twildh/quick-start) Addon for that. It is similar to Vivaldi although not completely the same as Vivaldi, but very close to it. You may give it a try.

**Comment by summoner001** [comment]
It seems quickstart is nice speed dial. But unfortunatelly the development has stopped or abandoned. If it were to be implemented, it would be perfectly usable with a little tweaking.

**Reply by SystemError12** [reply]
Did you try [Yet another speed dial](https://addons.mozilla.org/en-US/firefox/addon/yet-another-speed-dial/)? It is a little closer to present Vivaldi with a Tabs or Folder Menu (on the top left, where Vivaldi has it in the top Middle) and support of wallpaper. Quick-Start seems to be missing both.. 
The last update was 1 year ago; in comparison quickstart was 4 years ago. So it is a little more recent. Even though 1 year is already quite long..

**Reply by tasinone** [reply]
But you have to add Bookmarks separately in Yet another speed dial while Quick Start shows bookmarks from the browser's bookmark. So we don't have to worry about syncing and adding bookmarks again.

**Reply by SystemError12** [reply]
Yes true, but it's not a big issue for me, since I just put the webpages of everyday usage on my startpage and keep the rest in general bookmarks. I would love to manage both speed dial and bookmarks together and  properly integrated in the browser (and properly designed as well). That's why I made this Feature Request. 
But YASD is the closest alternative at least.
